### PR TITLE
fix(cirriculum): Expand instructions for Chai Assertion challenges

### LINKED ISSUE #35694: Fixing test descriptions in Quality Assurance and Testing with Chai
The test descriptions in the `Quality Assurance and Testing with Chai` section are unclear, vague and grammatically incorrect. One example can been seen [here](https://github.com/freeCodeCamp/freeCodeCamp/pull/35471#discussion_r268825980).

These test descriptions need to be fixed to make more sense

### PR DESCRIPTION
**Pull request by ranched**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #35694 

I added some basic language around the instructions to make them sentences and tried to standardize formatting across challenges in the section. If more detail is preferred I can add more.

### CODE REVIEW

File: curriculum/challenges/english/06-information-security-and-quality-assurance/quality-assurance-and-testing-with-chai/assert-deep-equality-with-.deepequal-and-.notdeepequal.english.md [hunk 1]
```diff
@@ -7,12 +7,12 @@ challengeType: 2
...
 <section id='description'>
 As a reminder, this project is being built upon the following starter project on <a href='https://glitch.com/#!/import/github/freeCodeCamp/boilerplate-mochachai/'>Glitch</a>, or cloned from <a href='https://github.com/freeCodeCamp/boilerplate-mochachai/'>GitHub</a>.
-<code>.deepEqual()</code>, <code>.notDeepEqual()</code>.
-<code>.deepEqual()</code> asserts that two object are deep equal.
 </section>
 
 ## Instructions
 <section id='instructions'>
+Use <code>assert.deepEqual()</code> or <code>assert.notDeepEqual()</code> to make the tests pass.
+<code>deepEqual()</code> asserts that two object are deep equal.
```
Comment (RandellDawson) [inline curriculum/challenges/english/06-information-security-and-quality-assurance/quality-assurance-and-testing-with-chai/assert-deep-equality-with-.deepequal-and-.notdeepequal.english.md]: This line seems for like it belongs in the Description and not the instruction.

File: curriculum/challenges/english/06-information-security-and-quality-assurance/quality-assurance-and-testing-with-chai/use-the-triple-equals-to-assert-strict-equality.english.md [hunk 2]
```diff
@@ -7,12 +7,12 @@ challengeType: 2
...
 <section id='description'>
 As a reminder, this project is being built upon the following starter project on <a href='https://glitch.com/#!/import/github/freeCodeCamp/boilerplate-mochachai/'>Glitch</a>, or cloned from <a href='https://github.com/freeCodeCamp/boilerplate-mochachai/'>GitHub</a>.
-<code>.strictEqual()</code>, <code>.notStrictEqual()</code>.
-<code>.strictEqual()</code> compares objects using <code>'==='</code>.
 </section>
 
 ## Instructions
 <section id='instructions'>
+Use <code>assert.strictEqual()</code> or <code>assert.notStrictEqual()</code> to make the tests pass.
+<code>strictEqual()</code> compares objects using <code>'==='</code>.
```
Comment (thecodingaviator) [inline curriculum/challenges/english/06-information-security-and-quality-assurance/quality-assurance-and-testing-with-chai/use-the-triple-equals-to-assert-strict-equality.english.md]:
```suggestion
<code>strictEqual()</code> compares objects using <code>===</code>.
```

File: curriculum/challenges/english/06-information-security-and-quality-assurance/quality-assurance-and-testing-with-chai/use-the-double-equals-to-assert-equality.english.md [hunk 3]
```diff
@@ -7,12 +7,12 @@ challengeType: 2
...
 <section id='description'>
 As a reminder, this project is being built upon the following starter project on <a href='https://glitch.com/#!/import/github/freeCodeCamp/boilerplate-mochachai/'>Glitch</a>, or cloned from <a href='https://github.com/freeCodeCamp/boilerplate-mochachai/'>GitHub</a>.
-<code>.equal()</code>, <code>.notEqual()</code>.
-<code>.equal()</code> compares objects using <code>'=='</code>.
 </section>
 
 ## Instructions
 <section id='instructions'>
+Use <code>assert.equal()</code> or <code>assert.notEqual()</code> to make the tests pass.
+<code>equal()</code> compares objects using <code>'=='</code>.
```
Comment (thecodingaviator) [inline curriculum/challenges/english/06-information-security-and-quality-assurance/quality-assurance-and-testing-with-chai/use-the-double-equals-to-assert-equality.english.md]:
```suggestion
<code>equal()</code> compares objects using <code>==</code>.
```

File: curriculum/challenges/english/06-information-security-and-quality-assurance/quality-assurance-and-testing-with-chai/use-assert.isok-and-assert.isnotok.english.md [hunk 4]
```diff
@@ -8,12 +8,12 @@ challengeType: 2
 <section id='description'>
 As a reminder, this project is being built upon the following starter project on <a href='https://glitch.com/#!/import/github/freeCodeCamp/boilerplate-mochachai/'>Glitch</a>, or cloned from <a href='https://github.com/freeCodeCamp/boilerplate-mochachai/'>GitHub</a>.
 
+<code>isOk(truthy)</code> and <code>isNotOk(falsey)</code> will pass.
```
Comment (RandellDawson) [inline curriculum/challenges/english/06-information-security-and-quality-assurance/quality-assurance-and-testing-with-chai/use-assert.isok-and-assert.isnotok.english.md]: This sentence does not make sense on its own.

**Review by RandellDawson — CHANGES_REQUESTED** [review]
See my comments.

File: curriculum/challenges/english/06-information-security-and-quality-assurance/quality-assurance-and-testing-with-chai/use-assert.isok-and-assert.isnotok.english.md (hunk 4 above)
Comment (ranched) [reply curriculum/challenges/english/06-information-security-and-quality-assurance/quality-assurance-and-testing-with-chai/use-assert.isok-and-assert.isnotok.english.md]: I've changed these lines to provide a little more context.

**Comment by ranched** [comment]
@RandellDawson just wanted to check if all requested changes were made on this. If there's anything else needing done I'm happy to update.

**Review by RandellDawson — APPROVED** [review]

File: curriculum/challenges/english/06-information-security-and-quality-assurance/quality-assurance-and-testing-with-chai/test-for-truthiness.english.md [hunk 5]
```diff
@@ -8,13 +8,17 @@ challengeType: 2
 <section id='description'>
 As a reminder, this project is being built upon the following starter project on <a href='https://glitch.com/#!/import/github/freeCodeCamp/boilerplate-mochachai/'>Glitch</a>, or cloned from <a href='https://github.com/freeCodeCamp/boilerplate-mochachai/'>GitHub</a>.
 
+<code>isTrue()</code> will test for the boolean value true and <code>isNotTrue()</code> will pass when given anything but the boolean value of true.
+assert.isTrue(true, 'this will pass with the boolean value true');
```
Comment (thecodingaviator) [inline curriculum/challenges/english/06-information-security-and-quality-assurance/quality-assurance-and-testing-with-chai/test-for-truthiness.english.md]: Put these three in a `<blockquote>` tag

Comment (thecodingaviator) [reply curriculum/challenges/english/06-information-security-and-quality-assurance/quality-assurance-and-testing-with-chai/test-for-truthiness.english.md]: These three i.e. lines 12-14

File: curriculum/challenges/english/06-information-security-and-quality-assurance/quality-assurance-and-testing-with-chai/test-if-a-string-contains-a-substring.english.md [hunk 6]
```diff
@@ -7,12 +7,15 @@ challengeType: 2
 ## Description
 <section id='description'>
 As a reminder, this project is being built upon the following starter project on <a href='https://glitch.com/#!/import/github/freeCodeCamp/boilerplate-mochachai/'>Glitch</a>, or cloned from <a href='https://github.com/freeCodeCamp/boilerplate-mochachai/'>GitHub</a>.
-<code>#include (or #notInclude)</code> works for strings too!!
-It asserts that the actual string contains the expected substring.
+
+<code>include()</code> and <code>notInclude()</code> work for strings too!!
```
Comment (thecodingaviator) [inline curriculum/challenges/english/06-information-security-and-quality-assurance/quality-assurance-and-testing-with-chai/test-if-a-string-contains-a-substring.english.md]:
```suggestion
<code>include()</code> and <code>notInclude()</code> work for strings too!
```

File: curriculum/challenges/english/06-information-security-and-quality-assurance/quality-assurance-and-testing-with-chai/test-if-a-string-contains-a-substring.english.md [hunk 7]
```diff
@@ -7,12 +7,15 @@ challengeType: 2
...
+include() asserts that the actual string contains the expected substring.
```
Comment (thecodingaviator) [inline curriculum/challenges/english/06-information-security-and-quality-assurance/quality-assurance-and-testing-with-chai/test-if-a-string-contains-a-substring.english.md]:
```suggestion
<code>include()</code> asserts that the actual string contains the expected substring.
```

**Review by thecodingaviator — CHANGES_REQUESTED** [review]

**Review by thecodingaviator — APPROVED** [review]

**Comment by thecodingaviator** [comment]
Thanks for contributing to the freeCodeCamp repository. Congratulations on your first PR! 🎊 

Look forward to more contributions from you! ✈️

**Comment by ranched** [comment]
Thanks! @thecodingaviator @RandellDawson thanks for your guidance!

**Pull request merged**